src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
@ADmad Yes there wouldn't be traces anymore. I thought this was better than sometimes not getting an error response at all.